packages/form-builder/src/settings/email/template-options/index.tsx

```diff
@@ -0,0 +1,205 @@
+import {Button, Modal, TextControl} from "@wordpress/components";
+import {edit} from "@wordpress/icons";
+import {__} from "@wordpress/i18n";
+import {useEffect, useState} from "react";
+import TabPanel from "@givewp/form-builder/components/sidebar/TabPanel";
+import EmailTemplateSettings from "./settings";
+import CopyToClipboardButton from "./components/copy-to-clipboard-button";
+import {getFormBuilderData, getStorageData} from "@givewp/form-builder/common/getWindowData";
+import {useFormState} from "@givewp/form-builder/stores/form-state";
+import SendPreviewEmail from "@givewp/form-builder/settings/email/template-options/components/send-preview-email";
+
+export default () => {
+
+    const [ isOpen, setOpen ] = useState<boolean>( false );
+    const openModal = () => setOpen( true );
+    const closeModal = () => setOpen( false );
+
+    const [ selectedTab, setSelectedTab ] = useState<string>();
+
+    const [ showPreview, setShowPreview ] = useState<boolean>( false );
+
+    const {templateTags, emailNotifications} = getFormBuilderData();
+
+    const EmailPreviewContent = ({emailType}) => {
+
+        const [ previewHtml, setPreviewHtml ] = useState<string>(null);
+
+        const {settings: {emailTemplateOptions, emailTemplate, emailLogo, emailFromName, emailFromEmail}} = useFormState();
```

Comment:
you'll want to update the type `FormState` to account for new settings

Reply:
These have been added to the `FormSettings` type, which is used in `FormState`.